examples/04_fuzzy_joining.py

```diff
@@ -289,8 +284,8 @@
     y="Happiness score",
     lowess=True,
 )
-fig.set_ylabel("Happiness index")
-fig.set_title("Does a country's legal rights strength lead to happiness?")
+fig.set_ylabel("Happiness Index")
+fig.set_title("Effect of a Country's Strength of Legal Rights on Happiness")
```

Comment:
if we're changing the title I rather like the "is x linked to y" formulation above because it is more clear that we are only talking about correlations

Reply:
Yeah fair enough. I will delete the wording changes. I just thought a question as a graph title looked a bit strange but happy to just leave them as is.